dot/core/service.go

```diff
@@ -526,47 +526,64 @@ func (s *Service) ProcessBlockRequestMessage(msg network.Message) error {
 	responseData := []*types.BlockData{}
 
 	for _, hash := range subchain {
-		data, err := s.blockState.GetBlockData(hash)
+		dataBody, err := s.blockState.GetBlockBody(hash)
```

Comment:
remove this, since block body is being retrieved at line 552

Reply:
updated.